projects/igniteui-angular/src/lib/directives/input/input.directive.ts

```diff
@@ -63,8 +64,6 @@ export enum IgxInputState {
     standalone: true
 })
 export class IgxInputDirective implements AfterViewInit, OnDestroy {
-    private static ngAcceptInputType_required: boolean | '';
-    private static ngAcceptInputType_disabled: boolean | '';
```

Comment:
Is this no longer needed so I can use the attribute without a value?

Reply:
Yes.

The changes to the boolean @Input properties (at least the simple ones) follow the pattern:

**Disabled state**

```html
<igx-switch disabled></igx-switch>
<igx-switch disabled=""></igx-switch>
<igx-switch disabled="true"></igx-switch>
```

**Enabled state**

```html
<igx-switch></igx-switch>
<igx-switch disabled="false"></igx-switch>
```